scripts/get_das_info.py

```diff
@@ -12,6 +12,35 @@
 import law
 
 
+def convert_to_desired_structure(data_dict):
+
+    # Create placeholders for nameing scheme
+    name = "PLACEHOLDER"
+    process = "PLACEHOLDER"
+
+    var_id = data_dict["dataset_id"]
+    var_key = data_dict["name"]
+    var_file = data_dict["nfiles"]
+    var_events = data_dict["nevents"]
+
+    # Create the function call as a string
+    # \042 ASCII code for quotation mark "
+    function_call = (
+        f"can.add_dataset(\n"
+        f"    name=\042{name}\042,\n"
+        f"    id={var_id},\n"
+        f"    processes=[procs.{process}],\n"
+        f"    keys=[\n"
+        f"        \042{var_key}\042,\n"
+        f"    ],\n"
+        f"    n_files={var_file},\n"
+        f"    n_events={var_events},\n"
+        f")"
+    )
+
+    return function_call
```

Comment:
I'm in favor of converting the data to a string that can be pasted :)

Just a suggestion for simplifying things:

```suggestion
def convert_to_desired_structure(data: dict) -> str:
    return f"""cpn.add_dataset(
    name="PLACEHOLDER",
    id={data['dataset_id']}
    processes=[procs.{PLACEHOLDER}],
    keys=[
        {data['name']},
    ],
    n_files={data['nfiles']},
    n_events={data['nevents']},
)"""
```

Reply:
I like the suggestion, that way we are not dependent on the ASCII characters. Implemented in updated PR.